bgpd/bgp_advertise.h

```diff
@@ -165,5 +165,6 @@ extern void bgp_advertise_delete(struct bgp_advertise_attr *baa,
 				 struct bgp_advertise *adv);
 extern void bgp_advertise_unintern(struct hash *hash,
 				   struct bgp_advertise_attr *baa);
+extern void baa_free(struct bgp_advertise_attr *baa);
```

Comment:
This name is awful.  bgp_advertise_attr_free?

Reply:
Do you suggest renaming this function?

Reply:
Fixed.